### ISSUE TITLE
"Could not find run number" error

### ISSUE THREAD

**Issue by pdeperio** [XENON1T/hax#43]
Seems to be a benign error running on MC ([command](https://github.com/XENON1T/processing/blob/master/montecarlo/run_sim.sh#L131), ```pax_version_policy = 'loose'```): 
```
Could not find run number for Xenon1T_TPC_Kr83m_96.000000_g4mc_NEST_Patch_pax, got exception <class 'IndexError'>: index 0 is out of bounds for axis 0 with size 0. Setting run number to 0.
```
Would be nice to suppress these to clean up logs and facilitate (real) error searching.

**Comment by JelleAalbers** [comment]
Yes, that's an expected message. Probably it would be best to add a dont_search_rundb option or something to suppress this.
